### PR TITLE
Bulldog mags in nukie uplink

### PR DESCRIPTION
**Pull request by JoeHammad1844**
## About the PR

Adds the lethal bulldog mag to the uplink for 2TC
Adds the beanbag bulldog mag to the uplink for 1TC

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: JoeHammad
- add: Nukies can now purchase lethal and beanbag bulldog magazines for 2TC and 1TC respectively.

### CODE REVIEW

**Comment by Errant-4** [comment]
If you want to open a PR before it's done, open it as a Draft

**Comment by JoeHammad1844** [comment]
good idea

**Comment by Errant-4** [comment]
Is this Suit and Tie thing an accidental addition from another change you are working on but did not meant to include?
The BoxSuitAndTie entity does not exist so it being in the Uplink throws an exception and the entire Pointless category gets broken

**Comment by JoeHammad1844** [comment]
hring

**Comment by JoeHammad1844** [comment]
:trollface: :trollface: :trollface: :trollface: :trollface: :trollface: :trollface: :trollface: :trollface: :trollface: :trollface: :trollface:

File: Resources/Prototypes/Catalog/uplink_catalog.yml
Comment (Errant-4) [inline Resources/Prototypes/Catalog/uplink_catalog.yml]: You have ripped out the Syndicate pAI along with your suit and tie

Comment (JoeHammad1844) [reply Resources/Prototypes/Catalog/uplink_catalog.yml]: fuck

**Comment by JoeHammad1844** [comment]
I have another uplink item i've made that ill merge later once the fate of this one and the suit and tie are decided

**Comment by Errant-4** [comment]
Are you still not checking diffs to the master version?

**Comment by Errant-4** [comment]
>  add: Nukies can now purchase lethal and beanbag bulldog magazines for 2TC and 1TC respectively

Please end changelog entries with a punctuation mark

**Comment by JoeHammad1844** [comment]
alright,done

**Comment by Errant-4** [comment]
> my face when error unrelated to the code i changed

Ignore it, it's not for you to deal with

**Comment by JoeHammad1844** [comment]
I know

File: Resources/Locale/en-US/store/uplink-catalog.ftl
Comment (Errant-4) [inline Resources/Locale/en-US/store/uplink-catalog.ftl:77]: Change the name to the actual exact name of the item
The description should not call it a box magazine either, and use a punctuation mark at the end of the description

File: Resources/Locale/en-US/store/uplink-catalog.ftl
Comment (Errant-4) [inline Resources/Locale/en-US/store/uplink-catalog.ftl:80]: Same here

**Review by Errant-4 — CHANGES_REQUESTED** [review]
Please fix the catalog entries as indicated, I don't see any other code issues. Then you will have to wait for/find an actual maintainer to review and merge it. Good luck

**Comment by JoeHammad1844** [comment]
oh i gotta make it compatible with the new nades now

File: Resources/Locale/en-US/store/uplink-catalog.ftl
Comment (Errant-4) [reply Resources/Locale/en-US/store/uplink-catalog.ftl]: Look up the actual item name from Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Magazines/shotgun.yml 
every other uplink entry uses the exact entity names

and you are still  describing it as a "box magazine", which it is not

**Comment by Errant-4** [comment]
I don't know how you are doing it, but you are somehow pushing empty commits. Which is on some level impressive, I guess, because I thought that's impossible

**Comment by Just-a-Unity-Dev** [comment]
> I don't know how you are doing it, but you are somehow pushing empty commits. Which is on some level impressive, I guess, because I thought that's impossible

git commit -m "rerun tests" --allow-empty

can be used to rerun tests if you don't have the permissions

**Comment by JoeHammad1844** [comment]
image
my face when error unrelated to changes made

**Comment by EmoGarbage404** [comment]
do not webedit PRs.

image

We state explicitly in our guidelines that you are required to test your changes and that you should not use the gh website to create PRs.

**Pull request closed without merging**
